charts/openfaas-cloud/templates/tls/auth-domain-cert.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
I forgot to say that "ingress shim" can create the certs, and we can do without the `Certificate` object going forward.

Reply:
(Like we do in k3sup)